### ISSUE TITLE
Rename cogutils repository to cogutil

### ISSUE THREAD

**Issue by ngeiswei** [opencog/cogutil#90]
The library is `libcogutil`, the include path is `opencog/util`, while this repo name is cogutils. The confusion was mentioned a couple of times on the mailing list. For these reasons I suggest to rename this repo cogutil.

**Comment by ngeiswei** [comment]
Done, you can erase from your brain that cogutil ever ended by s.

**Comment by linas** [comment]
brain erasure is easier said than done.  in english, plural nouns usually end with an s :-)

**Comment by ngeiswei** [comment]
I regret that I went that way, removing `s`, instead of adding `s` to the directory and the library. My rational at the time was that removing `s`would amount to less change than adding `s` everywhere else, but after that I realized that component and directory names overwhelming ends by `s` when plurality is involved.

ICH KANN INKONSISTENZ NICHT TOLERIEREN!!!

Seriously, I'm considering rewinding and adding `s` where it's missing. Maybe I'll keep that for a day I'm particularly sick and unproductive.
